src/modules/organisation/domain/organisation.service.ts

```diff
@@ -49,4 +70,96 @@ export class OrganisationService {
             items: organisations,
         };
     }
+
+    public async setAdministriertVon(parentId: string, childId: string): Promise<Result<void, DomainError>> {
+        const parentOrganisation: Option<OrganisationDo<true>> = await this.organisationRepo.findById(parentId);
```

Comment:
I think it advisable to introduce a exists(id) call to the repo and use that.
Just finding something, mapping it and then discarding just to assert its existence seems wasteful to me.

Also when introducing such a call I believe we could leverage similar simplifications on a repo level
Please ponder on this and investigate feasability